File: rb/test/exists_test.rb
# CorporateBullshitGenerator SDK exists test

require "minitest/autorun"
require_relative "../CorporateBullshitGenerator_sdk"

class ExistsTest < Minitest::Test
  def test_create_test_sdk
    testsdk = CorporateBullshitGeneratorSDK.test(nil, nil)
    assert !testsdk.nil?
  end
end

File: lua/test/exists_test.lua
-- CorporateBullshitGenerator SDK exists test

local sdk = require("corporate-bullshit-generator_sdk")

describe("CorporateBullshitGeneratorSDK", function()
  it("should create test SDK", function()
    local testsdk = sdk.test(nil, nil)
    assert.is_not_nil(testsdk)
  end)
end)
